### PR TITLE
Use the primitive boolean expression #7863

### PR DESCRIPTION
**Pull request by krotonet**
Use the primitive boolean expression #7863
imports and lines optimization

### Describe your changes :

I worked on this issue 7863 [,](https://github.com/open-metadata/OpenMetadata/issues/7863) it was open for a while, and seems quite simple. More readable, understandable code, use of primitive boolean, if statement

#
### Type of change :
- [x] Improvement
- [x] Documentation

#
### Checklist:
- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by krotonet** [comment]
need some help approving this pr

File: openmetadata-service/src/main/java/org/openmetadata/csv/CsvUtil.java [hunk 1]
```diff
@@ -13,9 +13,7 @@
 
 package org.openmetadata.csv;
 
-import static org.openmetadata.common.utils.CommonUtil.listOf;
-import static org.openmetadata.common.utils.CommonUtil.listOrEmpty;
-import static org.openmetadata.common.utils.CommonUtil.nullOrEmpty;
+import static org.openmetadata.common.utils.CommonUtil.*;
```
Comment (harshach) [inline openmetadata-service/src/main/java/org/openmetadata/csv/CsvUtil.java:16]: please do not use wild-card imports. can you keep the old code here and update the PR

**Review by harshach — CHANGES_REQUESTED** [review]

File: openmetadata-service/src/main/java/org/openmetadata/csv/CsvUtil.java (hunk 1 above)
Comment (krotonet) [reply openmetadata-service/src/main/java/org/openmetadata/csv/CsvUtil.java]: code updated..

**Comment by krotonet** [comment]
need the labels to run tests on changes please.

**Review by harshach — APPROVED** [review]

**Comment by krotonet** [comment]
hi @harshach, all tests passed except the java-checkstyle, what can be the problem?

**Comment by krotonet** [comment]
@harshach is there any thing i can do to pass this Java Checkstyle test?

**Comment by harshach** [comment]
@krotonet can you please run this command in the root folder of the project and commit any changes
```mvn googleformatter:format@reformat-sources ```

**Pull request closed without merging**